test/pages/Checkout.page.js

```diff
@@ -0,0 +1,31 @@
+class CheckoutYourInformation {
+   
+
+    get addFirstName() {  
```

Comment:
I'd rather name such properties like: 
```suggestion
    get firstName() {  
```
or `get firstNameEl() `
btw all other properties should be renamed accordingly